signingscript/docker.d/init_worker.sh

```diff
@@ -113,10 +111,14 @@ case $ENV in
       mobile)
         test_var_set 'AUTOGRAPH_FENIX_PASSWORD'
         test_var_set 'AUTOGRAPH_FENIX_USERNAME'
+        test_var_set 'AUTOGRAPH_FENIX_STAGE_PASSWORD'
+        test_var_set 'AUTOGRAPH_FENIX_STAGE_USERNAME'
```

Comment:
This will need to be changed to `AUTOGRAPH_STAGE_FENIX_PASSWORD` etc as well now.